### DISCUSSION TITLE
Migrate to ESM (+ Typescript)

### DISCUSSION THREAD

**Discussion by kaaax0815** [wg-easy/wg-easy#1018] [Ideas]
Imho Esm has a lot cleaner syntax and is the way to go in the future.
Also Typescript adds alot of information to the developer when maintaining this project.

Just wanted to suggest this and hear your opinion

**Comment by offizium-berndstorath** [comment]
Will be added with #1244

**Comment by tetuaoro** [comment]
im agree let's do it
